application.js

```diff
@@ -46,31 +48,85 @@ var sessionOptions = {
   }
 };
 
+/**
+ * Gets mongodb connection information from $fh.db
+ * Use Data Browser > Upgrade Database to get the connection information available
+ */
+function mongoInit(cb) {
```

Comment:
Might be good to move these to separate files instead of in the main application.js file.